class-3-branches-reset/commands.txt

```diff
@@ -1,6 +1,9 @@
 git remote add <alias_name> <.git url>
 git remote rm <alias_name>
+git remote -v 
 git branch
 git checkout -b <branch_name>
```

Comment:
@vikash123765 Note : u could used git branch <branch_name> to create the branch but u have to shift manually using checkout or switch